assets/components/leftMarginSection/leftMarginSection.scss

```diff
@@ -0,0 +1,25 @@
+.component-left-margin-section {
+  display: flex;
+
+  @include mq($from: tablet) {
+    &:before {
+      content: "";
+      flex-basis: 0;
+      flex-grow: 1;
+      // Fix for IE11, it doesn't treat inline elements as flex items.
+      display: block;
+    }
+  }
+}
+
+.component-left-margin-section__content {
+  flex-grow: 1;
```

Comment:
I think we were going to have a max width for this?

Reply:
I think that's up to design. I don't know what that width should be so I've not included it at the moment; it's hard to tell before the content gets added.